### DISCUSSION TITLE
Information and Documentation Structure: User Preferences

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#6005] [Archive]
## Which layout would you prefer?
> How would you structure **Information Theory** for yourself?

## Which layout would you offer? 
> How would you structure **Information Theory** for phone, tablet, and computer.

## Which layout is versatile?
> How would you structure **Information Theory** for Contributors and App users.
1. works for you personally,
2. works for beginners and up,
3. works for _comments_, _user documentations_, _knowledge base article_, _FAQ list_, or _blog post_,
4. works for search or reference,

**Comment by philleepflorence** [comment]
## Information Theory
Information theory is the scientific study of the quantification, storage, and communication of digital information. The field was fundamentally established by the works of Harry Nyquist and Ralph Hartley, in the 1920s, and Claude Shannon in the 1940s. The field is at the intersection of probability theory, statistics, computer science, statistical mechanics, information engineering, and electrical engineering.

<details>
<summary>Click to read more!</summary>
A key measure in information theory is entropy. Entropy quantifies the amount of uncertainty involved in the value of a random variable or the outcome of a random process. For example, identifying the outcome of a fair coin flip (with two equally likely outcomes) provides less information (lower entropy) than specifying the outcome from a roll of a die (with six equally likely outcomes). Some other important measures in information theory are mutual information, channel capacity, error exponents, and relative entropy. Important sub-fields of information theory include source coding, algorithmic complexity theory, algorithmic information theory, and information-theoretic security.

Applications of fundamental topics of information theory include lossless data compression (e.g. ZIP files), lossy data compression (e.g. MP3s and JPEGs), and channel coding (e.g. for DSL). Its impact has been crucial to the success of the Voyager missions to deep space, the invention of the compact disc, the feasibility of mobile phones and the development of the Internet. The theory has also found applications in other areas, including statistical inference, cryptography, neurobiology,[ perception, linguistics, the evolution and function of molecular codes (bioinformatics), thermal physics,[ quantum computing, black holes, information retrieval, intelligence gathering, plagiarism detection, pattern recognition, anomaly detection and even art creation.
</details>

**Comment by philleepflorence** [comment]
## Information Theory
Information theory is the scientific study of the quantification, storage, and communication of digital information. The field was fundamentally established by the works of Harry Nyquist and Ralph Hartley, in the 1920s, and Claude Shannon in the 1940s. The field is at the intersection of probability theory, statistics, computer science, statistical mechanics, information engineering, and electrical engineering.

A key measure in information theory is entropy. Entropy quantifies the amount of uncertainty involved in the value of a random variable or the outcome of a random process. For example, identifying the outcome of a fair coin flip (with two equally likely outcomes) provides less information (lower entropy) than specifying the outcome from a roll of a die (with six equally likely outcomes). Some other important measures in information theory are mutual information, channel capacity, error exponents, and relative entropy. Important sub-fields of information theory include source coding, algorithmic complexity theory, algorithmic information theory, and information-theoretic security.

Applications of fundamental topics of information theory include lossless data compression (e.g. ZIP files), lossy data compression (e.g. MP3s and JPEGs), and channel coding (e.g. for DSL). Its impact has been crucial to the success of the Voyager missions to deep space, the invention of the compact disc, the feasibility of mobile phones and the development of the Internet. The theory has also found applications in other areas, including statistical inference, cryptography, neurobiology,[ perception, linguistics, the evolution and function of molecular codes (bioinformatics), thermal physics,[ quantum computing, black holes, information retrieval, intelligence gathering, plagiarism detection, pattern recognition, anomaly detection and even art creation.

**Comment by philleepflorence** [comment]
## Information Theory

### Introduction
Information theory is the scientific study of the quantification, storage, and communication of digital information. 
<details>
<summary>Read more on concept...</summary>
The field was fundamentally established by the works of Harry Nyquist and Ralph Hartley, in the 1920s, and Claude Shannon in the 1940s. The field is at the intersection of probability theory, statistics, computer science, statistical mechanics, information engineering, and electrical engineering.
</details>

### Function
A key measure in information theory is entropy. Entropy quantifies the amount of uncertainty involved in the value of a random variable or the outcome of a random process. 
<details>
<summary>See examples of function...</summary>
For example, identifying the outcome of a fair coin flip (with two equally likely outcomes) provides less information (lower entropy) than specifying the outcome from a roll of a die (with six equally likely outcomes). Some other important measures in information theory are mutual information, channel capacity, error exponents, and relative entropy. Important sub-fields of information theory include source coding, algorithmic complexity theory, algorithmic information theory, and information-theoretic security.
</details>

### Application 
Applications of fundamental topics of information theory include lossless data compression (e.g. ZIP files), lossy data compression (e.g. MP3s and JPEGs), and channel coding (e.g. for DSL). 
<details>
<summary>See examples for application...</summary>
Its impact has been crucial to the success of the Voyager missions to deep space, the invention of the compact disc, the feasibility of mobile phones and the development of the Internet. The theory has also found applications in other areas, including statistical inference, cryptography, neurobiology,[ perception, linguistics, the evolution and function of molecular codes (bioinformatics), thermal physics,[ quantum computing, black holes, information retrieval, intelligence gathering, plagiarism detection, pattern recognition, anomaly detection and even art creation.
</details>

**Comment by benhaynes** [comment]
Any more insight into what you're trying to solve here? I'm a bit lost on where this would be implemented within the Docs.

**Reply by philleepflorence** [reply]
I already have some insight based on your response...

Choose an **answer** you like for **Information Theory**.
